azure-ts-appservice/index.ts

```diff
@@ -0,0 +1,104 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+import { signedBlobReadUrl } from "./sas";
+
+const name = 'azpulumi';
+
+const resourceGroup = new azure.core.ResourceGroup(`${name}-rg`, {
+        location: "West US 2",
+    });
+
+const resourceGroupArgs = {
+    resourceGroupName: resourceGroup.name,
+    location: resourceGroup.location,
+};
+
+const storageAccount = new azure.storage.Account(`${name}sa`, {
+    ...resourceGroupArgs,
+
+    accountKind: "StorageV2",
+    accountTier: "Standard",
+    accountReplicationType: "LRS",
+});
+
+
+const appServicePlan = new azure.appservice.Plan(`${name}-asp`, {
+    ...resourceGroupArgs,
+
+    kind: "App",
+
+    sku: {
+        tier: "Basic",
+        size: "B1",
+    },
+});
+
+const storageContainer = new azure.storage.Container(`${name}-c`, {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+    containerAccessType: "private",
+});
+
+const map: pulumi.asset.AssetMap = {};
+map["index.html"] = new pulumi.asset.StringAsset("<html><body><h1>Greetings from Azure App Service!</h1></body></html>");
```

Comment:
Was there any reason this could not be written as:

```typescript
const map: pulumi.asset.AssetMap = {
    "index.html": new pulumi.asset.StringAsset("<html><body><h1>Greetings from Azure App Service!</h1></body></html>"),
};
```

Reply:
I copy-pasted it from `azure-ts-function` sample, I'll change

Reply:
Got it :smile:.  We'll update the code in the other example as well.